src/blocks/modal.css

```diff
@@ -160,6 +160,46 @@
   cursor: none;
 }
 
```

Comment:
All modals should have ` font-family: "Poppins", Arial, Helvetica, sans-serif;`. Please, add it for the `.modal` styles in line 2 in `modal.css`.


```css

.modal {
  font-family: "Poppins", Arial, Helvetica, sans-serif;
  position: fixed;
  top: 0;
// and so on
```